docs/throttling.md

```diff
@@ -67,9 +70,61 @@ For more information and a complete list of features visit the documentation on
 # Enable system traffic throttling
 throttle 3gfast
 
-# Run Lighthouse with its own throttling disabled
-lighthouse --throttling-method=provided # ...
+# Run Lighthouse with its own network throttling disabled (while leaving CPU throttling)
+lighthouse --throttling-method=devtools \
+  --throttling.requestLatencyMs=0 \
+  --throttling.downloadThroughputKbps=0 \
+  --throttling.uploadThroughputKbps=0 \
+  https://example.com
 
 # Disable the traffic throttling once you see "Gathering trace"
 throttle --stop
 ```
+
+# CPU Throttling
+
+Lighthouse applies CPU throttling to emulate a mid-tier mobile device even when run on far more powerful desktop hardware.
+
+## Types of CPU Throttling
+
+Within web performance testing, there are two typical styles of CPU throttling:
+
+1. **_Simulated throttling_**, which Lighthouse uses by **default**, uses a simulation of a page load, based on the data observed in the initial unthrottled load. This approach makes it very fast. However, due to the imperfect nature of predicting alternate execution paths, there is inherent inaccuracy that is summarized in this doc: [Lighthouse Metric Variability and Accuracy](https://docs.google.com/document/d/1BqtL-nG53rxWOI5RO0pItSRPowZVnYJ_gBEQCJ5EeUE/edit). The TLDR: while it's fairly accurate for most circumstances, it suffers from edge cases and a deep investigation to performance should use _applied_ CPU throttling tools.
+1. **_Applied throttling_** , also called _`devtools` throttling_ in Lighthouse configuration. This method actually interrupts execution of CPU work at periodic intervals to emulate a slower processor. It is [fairly accurate](https://docs.google.com/document/d/1jGHeGjjjzfTAE2WHXipKF3aqwF2bFA6r0B877nFtBpc/edit) and much easier than obtaining target hardware.
```

Comment:
yeah I can adjust to mention it. I didn't really want to because unlike the net throttling case it's not fundamentally different AFAIK from DevTools approach just happening at a different layer, but it's worth saying this at least 👍

Reply:
```suggestion
1. **_Applied throttling_** , also called _`devtools` throttling_ in Lighthouse configuration. This method actually interrupts execution of CPU work at periodic intervals to emulate a slower processor. It is [fairly accurate](https://docs.google.com/document/d/1jGHeGjjjzfTAE2WHXipKF3aqwF2bFA6r0B877nFtBpc/edit) and much easier than obtaining target hardware. The same underlying principle can be used by [linux cgroups](https://www.kernel.org/doc/html/latest/scheduler/sched-bwc.html) to throttle any process, not just the browser. Other tools like [WebPageTest use applied CPU throttling](https://github.com/WPO-Foundation/wptagent/commit/f7fe0d6b5b01bd1b042a1fe3144c68a6bff846a6) offered by DevTools.
```